webapp/components/DeleteData/DeleteData.spec.js

```diff
@@ -68,15 +66,15 @@ describe('DeleteData.vue', () => {
       jest.clearAllMocks()
     })
 
-    it('defaults to deleteContributions to false', () => {
+    it('checkbox deleteContributions defaults be false', () => {
       expect(wrapper.vm.deleteContributions).toEqual(false)
     })
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T09:41:21Z" title="Wednesday, May 27th 2020, 11:41:21 am +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -93,7 +93,11 @@ describe('DeleteData.vue', () => {
         deleteAccountBtn = wrapper.find('[data-test="delete-button"]')
       })
 
-      it('if deleteEnabled is true and only deletes user by default', () => {
+      it('if deleteEnabled is true and only deletes user ', () => {
+        mocks.$t.mockImplementation(() => deleteContributionsMessage)
+        enableContributionDeletionCheckbox = wrapper.findAll('input[type="checkbox"]').at(0)
```


Like it is done above with: ☝🏼
`deleteAccountBtn = wrapper.find('[data-test="delete-button"]')`

Instead of this programers on the net recommend always use something like
`enableContributionDeletionCheckbox = wrapper.find('[data-test="contributions-deletion-checkbox"]')`
and set
`data-test="contributions-deletion-checkbox"` as property on the input @mattwr18 has found and I agree very much on.
And for similar calls below as well. 👇🏼

This will make the tests stable against any future changes on the checkboxes.

But just on this line you don’t need it anyways, because you don’t trigger a click here afterwards. 😄